### DISCUSSION TITLE
当时玩到这里的时候真的很震撼，真正体会到了烧鸡老师的作品是有多么牛逼。可以说基本没有其他二游可以超越（米家游戏除外）

### DISCUSSION THREAD

**Discussion by did1181** [share121/inter-knot#2913] [其他]
![Screenshot 2024-02-06 230439](https://github.com/user-attachments/assets/62c0cb28-22d1-4e17-9e9a-d2a21a0e0292)

**Comment by NiuBoss123** [comment]
有点绝对说实话，麻枝准的剧情也不差啊（指红烧天堂）
当然，这东西还是看个人，个人喜欢就算是好剧情，不喜欢的就不是力（迫真
<img src="https://bbs-static.miyoushe.com/static/2023/10/07/af28fb66c5701a3dbcd65800211226cc_3156555622246650632.png" alt="卡琳_星星眼" width="75" >

**Comment by user51534** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >打完看看..2.3剧情挺一般的，三次死亡死的有点唯心主义，乐子人也没啥大乐子，强行搞个女友形象也很奇怪

**Reply by NiuBoss123** [reply]
收尾收不好好像是烧鸡的老问题了（简单来说是会留坑，参考崩三那边的情况），不过不妨碍烧鸡写的剧情还是很能调动玩家情绪的，就是可能要帆船来擦屁股什么的
